src/main/java/com/booksaw/betterTeams/events/ChatManagement.java

```diff
@@ -34,10 +34,6 @@ public static void enable() {
 	@EventHandler(priority = EventPriority.HIGH)
 	public void onChat(AsyncPlayerChatEvent event) {
 
-		if (event.isCancelled()) {
```

Comment:
Is there a reason you have removed this? 
This would mean that we would ignore up-stream plugins cancelling the event which is generally not good practice. 

It may be better instead to add the ignoreCancelled flag to the `@EventHandler` tag (as it defaults to false at present the event will be called when it has been cancelled upstream)
See docs here: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/EventHandler.html

Reply:
You're absolutely right. I had interpreted the ignoreCancelled flag as setting the handler to ignore whether an event had been cancelled. I thought that the default being false meant that the handler *would not* be called for previously cancelled events. I recognize now that my interpretation was wrong and that the flag means to set the handler to ignore cancelled events which does make more sense.